### ISSUE TITLE
add webpagetest

### ISSUE THREAD

**Issue by phildotdev** [fephil/foley#38]
This is quite a big job. Consider:
- Travis
- Site needs to be public (maybe a variable in foley.json?)
- Gulp integration (eg http://seesparkbox.com/foundry/automating_performance_testing_with_gulp)

**Comment by furzeface** [comment]
I'll start this off by implementing the perf tests I've been working on if you want?

**Comment by phildotdev** [comment]
Oh man that would be amazing! I'm blocked by life at the moment but I've got Sunday pencilled in

**Comment by furzeface** [comment]
Cool I'll make a start on that this weekend. I'm away so won't come work with you but have my laptop with me anyway.

**Comment by phildotdev** [comment]
This is on hold as is might be better to suggest an external tool for this.
